tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java

```diff
@@ -251,6 +251,10 @@ public static String getContainerDockerKey() {
   // Job types that we will short circuit when it failed
   public static final String STOP_ON_FAILURE_JOBTYPES = TONY_APPLICATION_PREFIX + "stop.on.failure.jobtypes";
 
+  // Tony configuration to return failure when a worker failed
+  public static final String FAIL_ON_WORKER_FAILURE_ENABLED = TONY_APPLICATION_PREFIX + "fail.on.worker.failure.enabled";
```

Comment:
I think we want to use the '.' delimiter for creating a hierarchy. In this case it would be better to follow the convention: "fail-on-worker-failure-enabled".

Reply:
ok